.circleci/config.yml

```diff
@@ -881,18 +892,6 @@ jobs:
           name: "Test"
           command: cond_spot_run_tests end-to-end e2e_p2p_network.test.ts
 
-  # TODO(AD): Reenable after #1367
-  # e2e-uniswap-sandbox:
```

Comment:
I guess this canary is the route we chose for uniswap sandbox tests?

Reply:
It's just that the canary effectively duplicates this test.